README.md

```diff
@@ -12,6 +12,7 @@ In this project, you will build a simple HTML list of To Do tasks. The list will
 # Link to the result of the website
  [ To-do List](https://khanetmoi.github.io/To-Do-list/)
 
+
```

Comment:
- [ ] [optional] I would suggest linking your issues page and MIT license in the README file, It will make your README file looks more professional.